meerk40t/ruida/device.py

```diff
@@ -520,7 +525,7 @@ def write(self, data):
         for array in self.parse_commands(data):
             try:
                 self.process(array)
-            except CommandSyntaxError:
+            except RuidaCommandError:
```

Comment:
I believe I caught this somewhere and wanted the type to be SyntaxError for that error flow. It works fine in RuidaCommandError but must be caught in the "write" section of the code.

Reply:
For the same reasons as changing console command syntax exceptions, we should not use the Python SyntaxError exception for this either. I am unclear what (if anything) you are wanting me to do here?